compiler/commands.nim

```diff
@@ -396,6 +396,8 @@ proc processSwitch*(switch, arg: string, pass: TCmdLinePass, info: TLineInfo;
   var
     key, val: string
   case switch.normalize
+  of "input", "i": # PRTEMP
+    expectArg(conf, switch, arg, pass, info)
```

Comment:
@timotheecour Please check/review your PR to find things like this, before "threatening" to merge it.

Reply:
https://github.com/nim-lang/Nim/pull/15642#discussion_r510352403